sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -328,15 +303,14 @@ getClient().send(chatMessage: message, chatThreadId: threadId) { result, _ in
 
 #### Get a message
 
-Use the `getChatMessage` method to retrieve a message in a thread.
+Use the `get` method of `ChatThreadClient` to retrieve a message in a thread.
 
-- `chatThreadId` is the unique ID of the thread.
 - `chatMessageId` is the unique ID of the message.
 
 `ChatMessage` is the response returned from getting a message.
 
 ```swift
-client.getChatMessage(chatThreadId: threadId, chatMessageId: messageId) { result, _ in
+chatThreadClient.getChatMessage(chatMessageId: messageId) { result, _ in
```

Comment:
Same thing here, either `get(message:)` or `getMessage(withId:)`

Reply:
The more I see this, the more I prefer the first option. We don't use `getContainer(withName: String,...)` in Storage, so I'm not sure why we would here. In the case of Storage, the identifier of a container is a name--in the case of Chat, it's an id.